### DISCUSSION TITLE
Variants archived?

### DISCUSSION THREAD

**Discussion by jarnold231** [owid/covid-19-data#2249] [Q&A] [answered]
I saw that the variant information has been archived - is this likely to return, or is that dataset now finished?

**Comment by edomt** [comment, marked as answer]
Hi @jarnold231 

The data on variants and sequencing is indeed no longer available to download.
It is published by GISAID under a license that doesn't allow us to redistribute it.
Please visit [the data publisher's website](https://www.gisaid.org/) for more details. You may want to register an account there if you're really interested in using this data.

**Reply by jarnold231** [reply]
Thanks for the quick response, thats a shame they dont let you republish it - I found your .csv file far easier to work with than directly compiling it from Covariants github. I shall take a look at the GISAID site and see if I can access it there.

**Reply by augh62** [reply]
> Hi @jarnold231
> 
> The data on variants and sequencing is indeed no longer available to download. It is published by GISAID under a license that doesn't allow us to redistribute it. Please visit [the data publisher's website](https://www.gisaid.org/) for more details. You may want to register an account there if you're really interested in using this data.

I understand the license issue but, in principle, it should be possible (without breaking the license) to use the owid/covid-19 code to download data adn put in the same format you redistributed it. Right?

**Reply by erdemah** [reply]
Hi, I've been looking for covid variant cases by country. I ended up being here. Is there anywhere else that I could find a dataset related to this?
Thanks!

**Reply by edomt** [reply]
@erdemah Your best bet is probably https://covariants.org/

**Reply by msh18** [reply]
@erdemah @edomt  Hello! I am also looking for a historical dataset of daily covid variant cases by country. All websites refer to GISAID, but I cannot register here, button Register is not active, section Contacts is empty. Do you know any other data sources or tutorials how to register on GISAID

**Reply by erdemah** [reply]
thanks @edomt. I manually labelled the variants in my dataset from your dashboard. Surprising that this dataset is not publicly available.

**Comment by Kingrider3** [comment]
Okay
